core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.html

```diff
@@ -49,4 +49,4 @@
 </ng-template>
 
 <p-confirmDialog [key]="confirmDialogKey" [style]="{ width: '30vw' }" />
-<p-confirmPopup></p-confirmPopup>
+<p-confirmPopup dotRemoveConfirmPopupWithEscape></p-confirmPopup>
```

Comment:
I am guessing the `closeOnEscape` input is not working?

Reply:
No, I added it when you created the popup with the service but is not working. sad.